transport_nantes/transport_nantes/settings.py

```diff
@@ -46,6 +46,7 @@
     'topicblog.apps.TopicBlogConfig',
     'dashboard.apps.DashboardConfig',
     'utm.apps.UtmConfig',
+    'gom_jabbar.apps.GomJabbarConfig',
```

Comment:
At this point of review (4bea2cfd3c890417951e3c788f09e742f0361173), it's not clear why we are duplicating the UTM-now-Visit app, you mention it's staging for another process but I think the commit comment could be clearer : 
- While I like the nerd reference to Dune, it's not very obvious at first glance that it aims to differentiate humans for robots unless you googled what gom_jabbar is
- I wish the the problem you're trying to solve doing this was specified in the commit message, because right now, I only know that this is a preliminary step to another change.